src/biokbase/narrative/tests/narrative_mock/mockclients.py

```diff
@@ -81,13 +83,13 @@ def ver(self):
     def get_workspace_info(self, params):
         """
         Some magic workspace ids.
-        12345 - the standard one.
+        12345 (WSID_STANDARD) - the standard one.
```

Comment:
Lol y u use 12345 for several different things and hardcode it

Reply:
😖

Reply:
🤦